ruff.toml

```diff
@@ -6,4 +6,5 @@ ignore = [
     "D104",  # D104 Missing docstring in public package
     "PLR0913", # PLR0913 Too many arguments in function definition
     "D100", # D100 Missing docstring in public module
+    "I001", # Import block is un-sorted or un-formatted
```

Comment:
This appears to be a necessary evil - the license headers are triggering the sorted imports rule.

Reply:
That's weird. It works fine with the headers beforehand...